Documentation/BuildInstructions.md

```diff
@@ -13,7 +13,7 @@ Optional: `fuse2fs` for [building images without root](https://github.com/Sereni
 
 #### GCC 12 or Clang 15
 
-A host compiler that supports C++20 features is required for building host tools, the newer the better. Tested versions include gcc-12 and clang-15.
+A host compiler that supports C++23 features is required for building host tools, the newer the better. Tested versions include gcc-12 and clang-15.
```

Comment:
mind the version numbers (anywhere else we mention supported/recommended compiler versions?)

Reply:
Those compiler versions support -std=c++2b, which cmake will pass along. Once we start actually using new features, we can revisit the minimums